src/components/DataTable/DataTable.vue

```diff
@@ -228,16 +228,21 @@ export default defineComponent({
 <style scoped>
 
 .data-table {
+  --datatable-text-color: #465968;
+  --datatable-border-color: #e8e8e8;
+  --datatable-background-hover-color: #fcfdfe;
+  --datatable-icon-color: #d2d9df;
```

Comment:
I believe all of these colors have sass variables for the broader miter color palette. I don't think we want to define a pallet for this component specifically. Maybe just use the sass variables for now. At some point I want to create custom properties for all those but I don't think they exist yet.

Reply:
Okay, I'll update those now